js/components/util/files.ts

```diff
@@ -122,55 +110,39 @@ export const loadHttpFile = async (
     if (url.endsWith('.json')) {
         return loadJSON(disk2, number, url);
     } else {
-        return new Promise((resolve, reject) => {
-            fetch(url).then(function (response: Response) {
-                if (response.ok) {
-                    const reader = response!.body!.getReader();
-                    let received = 0;
-                    const chunks: Uint8Array[] = [];
-                    // const contentLength = parseInt(response.headers.get('content-length')!, 10);
+        const response = await fetch(url);
```

Comment:
Since the `if` above returns, can this `else` block just be removed and its contents un-indented?